db2_for_machine_learning_samples/Db2 Hackathon - Learnware/README.md

```diff
@@ -0,0 +1,37 @@
+# Db2 Hackathon - Learnware
+
+Our recent Db2 hackathon demonstrate the versatile powers of Db2 combined with machine learning, web development, and blochaine. Through this hackathon, we had three amazing winners that used our products in intriguing ways that we would like to showcase. One of the three winners was ***Kuro Souza***. Kuro's project was called `Learnware`. This project used open source data in order to predict student performance. The steps below will show how to re-create Kuro's work for you to try out!
+
+
+Original Data Files - https://www.kaggle.com/rocki37/open-university-learning-analytics-dataset
+
+
+Live Version of Learnware - http://sparge1.fyre.ibm.com:8888/notebooks/notebook/learnware.ipynb
```

Comment:
This is internal-only and won't be broadly useful to external consumers of this sample.  Should we change or remove this?

Reply:
Yes, I think you are right. I took it out of the README.md now. It should be good to go